### ISSUE TITLE
nchan_redis_storage_mode backup does not working

### ISSUE THREAD

**Issue by Stas-Ghost** [slact/nchan#419]
## Steps to reproduce:
1. Clone repo https://github.com/Stas-Ghost/nchan-docker
All versions are stored in Dockerfile
2. Add nchan_redis_storage_mode backup; to nginx.vh.default.conf
```
   nchan_redis_namespace nchan;
   nchan_access_control_allow_origin $http_origin;
 + nchan_redis_storage_mode backup; 
```
3. Run docker-compose up
4. Execute
```
curl --request POST --data "test message" http://127.0.0.1:8081/pub/1
```
5. Nchan failed
```
nchan_1  | 172.20.0.1 - - [23/Nov/2017:16:10:48 +0000] "POST /pub/1; HTTP/1.1" 404 169 "-" "curl/7.56.1" "-"
nchan_1  | Assertion failed: !msg_in_shm (/usr/src/nchan-1.1.10/src/store/memory/memstore.c: nchan_store_publish_message_to_single_channel_id: 3200)
nchan_1  | 2017/11/23 16:10:51 [alert] 1#1: worker process 9 exited on signal 6 (core dumped)
```

## Problem
Config option
nchan_redis_storage_mode backup;
does not work

## Expected behaviour
Config option
nchan_redis_storage_mode backup;
works

**Comment by slact** [comment]
Fixed in 6462d102

**Comment by slact** [comment]
Fix released in 1.1.11
